src/facade/dragonfly_connection.cc

```diff
@@ -1207,6 +1207,10 @@ Connection::ParserStatus Connection::ParseRedis(unsigned max_busy_cycles) {
 
       DispatchSingle(has_more, dispatch_sync, dispatch_async);
     }
+    if (result != RedisParser::OK && result != RedisParser::INPUT_PENDING) {
```

Comment:
Please add a comment that we do not expect that a replica sends an invalid command so we log if it happens.